### PR TITLE
Pmgrove/organisation sample app

### PR DESCRIPTION
**Pull request by pmgrove**
:koala:

New organisation sidebar sample app

/cc @zendesk/vegemite, @zendesk/wombat 

### References
 - Jira link: n/a

### Risks
 - None

### CODE REVIEW

File: basic_organization_sample_app/translations/en.json [hunk 1]
```diff
@@ -0,0 +1,56 @@
+{
+  "app": {
+    "package": "basic_user_sample",
```
Comment (sandlerr) [inline basic_organization_sample_app/translations/en.json]: user -> org

File: basic_organization_sample_app/translations/en.json [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+    "title": "loading placeholder"
+  },
+
+  "fetch": {
+    "done": {
+      "value": "Good",
+      "title": "fetch success"
+    },
+    "fail": {
+      "value": "failed to fecth information from the server",
```
Comment (sandlerr) [inline basic_organization_sample_app/translations/en.json]: fecth should be fetch

File: hello_world_sample_app/templates/layout.hdbs [hunk 3]
```diff
@@ -3,4 +3,7 @@
   <h3>{{setting "name"}}</h3>
 </header>
 <section data-main/>
+<footer>
+	hello world
```
Comment (sandlerr) [inline hello_world_sample_app/templates/layout.hdbs]: did you mean to include this?

File: basic_organization_sample_app/app.css [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+  color: #fff;
+  text-shadow: 1px 1px #777;
+}
+
+.colored:nth-child(2n) {
+  background-color: #999;
+}
+
+.colored:nth-child(2n+1) {
+  background-color: rgb(149, 190, 34);
```
Comment (sandlerr) [inline basic_organization_sample_app/app.css]: I think our convention is hex everywhere unless you need alpha

File: basic_organization_sample_app/translations/en.json (hunk 2 above)
Comment (pmgrove) [inline basic_organization_sample_app/translations/en.json]: I will fix this up in the user sample app as well :)

File: hello_world_sample_app/templates/layout.hdbs (hunk 3 above)
Comment (pmgrove) [inline hello_world_sample_app/templates/layout.hdbs]: I will remove this from the user sample app as well

File: basic_organization_sample_app/app.css (hunk 4 above)
Comment (pmgrove) [inline basic_organization_sample_app/app.css]: This CSS came from the user sample app so am happy to convert both to hex.

**Comment by pmgrove** [comment]
@sandlerr :haircut:

**Comment by sandlerr** [comment]
:+1: I don't mind if the footer stays or goes

**Comment by danielbreves** [comment]
Nice :+1:

**Pull request merged**